src/set_theory/cardinal/basic.lean

```diff
@@ -624,13 +619,13 @@ end⟩
 /-- The indexed supremum of cardinals is the smallest cardinal above
   everything in the family. -/
 def sup {ι : Type u} (f : ι → cardinal.{max u v}) : cardinal :=
-Sup (set.range f)
+supr f
```

Comment:
will you remove this in a subsequent PR?

Reply:
I hope to, though that will require another substantial refactor.